numbat/src/lib.rs

```diff
@@ -253,6 +256,106 @@ impl Context {
         words.into_iter().filter(move |w| w.starts_with(word_part))
     }
 
+    pub fn print_info_for_keyword(&mut self, keyword: &str) -> Markup {
+        if keyword.is_empty() {
+            return m::text("Usage: info <unit or variable>");
+        }
+        let reg = self.interpreter.get_unit_registry();
+
+        if let PrefixParserResult::UnitIdentifier(_span, prefix, _, full_name) =
+            self.prefix_transformer.prefix_parser.parse(keyword)
+        {
+            if let Some(md) = reg
+                .inner
+                .get_base_representation_for_name(&full_name)
+                .ok()
+                .map(|(_, md)| md)
+            {
+                let mut help =
+                    m::text("Unit: ") + m::unit(md.name.as_deref().unwrap_or(keyword)) + m::nl();
+                if let Some(url) = &md.url {
+                    help += m::string(url) + m::nl();
+                }
+                if !md.aliases.is_empty() {
+                    help += m::text("Aliases: ")
+                        + m::text(
+                            md.aliases
+                                .iter()
+                                .map(|(x, _)| x.as_str())
+                                .collect::<Vec<_>>()
+                                .join(", "),
+                        )
+                        + m::nl();
+                }
+
+                help += m::text("A unit of [") + md.readable_type + m::text("]") + m::nl();
```

Comment:
Maybe we could special-case `Scalar`/dimensionless quantities here. For example: `info million` prints:
```
A unit of [Scalar]
```
Would be great to print sth. like "A dimensionless unit".

Reply:
do you see any value in keeping the word "Scalar" just to make sure users are familiar with it?   For example, the output of `type(5)` is `Scalar` and `Scalar` is used when definition functions that only scalar values.  Maybe we could do:

```
A dimensionless unit ([Scalar])
```

Reply:
Yeah, sounds good